chunks/tool-mongodb/Dockerfile

```diff
@@ -6,11 +6,20 @@ USER gitpod
 # Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
 ENV TRIGGER_REBUILD=1
 
+# Install MongoDB Shell aka MongoSH (was part of Mongo 5, but, is separate now)
+RUN mkdir -p /tmp/mongosh && \
+    cd /tmp/mongosh && \
+    wget -qOmongosh.tgz https://downloads.mongodb.com/compass/mongosh-1.8.0-linux-x64.tgz && \
```

Comment:
I tried looking for automation of `mongosh-*` version. But doesn't find any, on their [downloads page](https://www.mongodb.com/try/download/shell) & [docs](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install). Let's keep it, this way only & will patch the changes in future.